bot/cogs/color.py

```diff
@@ -677,6 +683,234 @@ async def sync_color(self, ctx: commands.Context) -> None:
+                "tertiary_color": None,
+            },
+            "permissions": "0",
+        }
+
+        async with aiohttp.ClientSession() as session:
+            async with session.post(url, json=payload, headers=headers) as resp:
+                if resp.status != 200:
+                    return await interaction.followup.send(
+                        f"❌ Failed to create gradient role. Discord API returned status {resp.status}.",
+                        ephemeral=True,
+                    )
+                data = await resp.json()
+                created_role = interaction.guild.get_role(int(data["id"]))
+                if not created_role:
+                    return await interaction.followup.send(
+                        "❌ Role was created but could not be found in the guild.",
+                        ephemeral=True,
+                    )
+        logger.info(f'NEW ROLE HAS BEEN CREATED SUCCESSFULLY. ROLE ID: {created_role.id}')
+
+        async with self.db_pool.acquire() as conn:
+            await conn.execute(
+                'INSERT INTO custom_role (role_id, owner_discord_id) VALUES ($1, $2)',
+                created_role.id,
+                role_owner.id,
+            )
+
+        self.custom_role_data[created_role.id] = role_owner.id
+        self.custom_role_data_list = list(self.custom_role_data.keys())
+
+        # Put recent created role under boundary role
+        boundary_role = interaction.guild.get_role(CustomRoleConfig.BOUNDARY_ROLE_ID)
+        while created_role.position != boundary_role.position - 1:
+            created_role = await created_role.edit(position=boundary_role.position - 1)
+
+        # Use created role immediately
+        role_being_used = get_current_custom_role_on_user(self, interaction.guild, role_owner)
+        if role_being_used:
+            await role_owner.remove_roles(role_being_used)
+        await role_owner.add_roles(created_role)
+
+        self.cache['color-list'] = None
+
+        embed = discord.Embed(
+            color=valid_color_primary,
+            description=f'✅ Successfully created and attached role: **{created_role.name}**.',
+        )
+        embed.add_field(
+            name="What to do next?",
+            value=(
+                "- Use </warnet-color list:1159052621177425951> to check list of all available custom roles\n"
+                "- Use </warnet-color set:1159052621177425951> to use any custom  role, or\n"
+                "- Use </warnet-color icon:1159052621177425951> to attach custom icon on your custom role (read <#822872937161162795> for instruction), or\n"
+                "- Use </warnet-color remove:1159052621177425951> to take off your current custom role, or\n"
+                "- Use </warnet-color edit hex:1159052621177425951> or </warnet-color edit rgb:1159052621177425951> to edit your created custom role"
+            ),
+        )
+
+        await interaction.followup.send(embed=embed)
```

Comment:
For now add an alert message on the embed footer to let them know this is unstable